scripts/class.js

```diff
@@ -75,7 +71,6 @@ function display() {
             `;
   });
 }
-// add event listener to the form
 bookForm.addEventListener('submit', (e) => {
   e.preventDefault();
   const newBook = new Book(bookTitle.value, bookAuthor.value);
```

Comment:
- [ ] Adding books to the collection is not working, Book is also not being stored in local storage. Please fix this issue and make sure the users are able to add and delete books from the collection.